app/assets/javascripts/application.js

```diff
@@ -31,7 +31,6 @@
 //= require scp-igv
 //= require scp-ideogram
 //= require scp-dot-plot
-//= require react_rails/bulkDownloadClipboard
```

Comment:
Let's restore this `require` after merging #389.  I mistakenly omitted a file in #387, then [added it](https://github.com/broadinstitute/single_cell_portal_core/pull/389/files#diff-644c19332e6dfb234e3eaf974eff11cfR1) in #389, which fixes the page load error that likely prompted you to comment this out.

It's needed for Bulk Download to work in the study detail page.

Reply:
Yeah, I had to take it out because it was throwing an error preventing the asset pipeline from rendering and thus blocking development.  I figured that it would get restored in another PR.